packages/web-twig/src/Resources/components/Modal/README.md

```diff
@@ -1,12 +1,241 @@
 # Modal
 
+## Composed Modal (DEPRECATED)
+
 This is Twig implementation of the [Modal] component.
 
+Modal is a composition of several subcomponents:
+
+- [ModalComposed](#modalcomposed)
+  - [ModalDialog](#modaldialog)
+  - [ModalHeader](#modalheader)
```

Comment:
The Header, Body, and Footer should not be on the same level as Dialog, should they?